src/internal_events/file.rs

```diff
@@ -270,6 +288,10 @@ impl FileSourceInternalEvents for FileSourceInternalEventsEmitter {
         });
     }
 
+    fn emit_file_open(&self, path: &Path, since: Instant) {
```

Comment:
I think we should call this `emit_file_opened`. I presume the name is like this because it's a gauge - but it's sort of an implementation detail, and at this level it's an event.

Reply:
As you said, because it's a gauge, but that's a part, the other is to differentiate it more from act of opening a file, for which we emit two other events `emit_file_added` and `emit_file_resumed`, `emit_file_opened` would make it even less clear. Also we are emitting this event every 1~2 sec for each file while they are open, so using present of `open`.

Reply:
I see, but seems even more confusing as is if it's the case.
So, we are emitting these events throughout the continuous process, right? Isn't there a better way of capturing that in the event name? Maybe something like `emit_file_is_still_open`? Or simply `emit_file_is_open`?

Reply:
True. 

Haven't seen `is` used in other events so what about, `fn emit_file_time_open`, change Instant to Duration, and rename `FileOpen` accordingly? In that case I would also like to rename the metric in this style, but this will need to be passed by Ben.